src/xo-app/backup/new/index.js

```diff
@@ -535,6 +540,10 @@ export default class New extends Component {
                     />
                   </fieldset>
                   <fieldset className='form-group'>
+                    <label>{_('jobTimeoutPlaceHolder')}</label>
+                    <input type='number' onChange={this.linkState('timeout')} value={timeout} className='form-control' />
```

Comment:
Use `state.timeout` instead of destructuring, let's try to keep those concerns as local as possible.